From: Duncan Eastoe <duncan.eastoe@att.com>
Date: Thu, 10 Dec 2020 16:57:10 +0000
Subject: frr-reload.py: treat "fpm" as single line context

Currently a line such as `fpm address 127.0.0.1` can trigger a cascade
of reload failures as it is treated as the start of a multi line
context.
---
 tools/frr-reload.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tools/frr-reload.py b/tools/frr-reload.py
index 88a40d8..810ba11 100755
--- a/tools/frr-reload.py
+++ b/tools/frr-reload.py
@@ -507,6 +507,7 @@ end
                                 "domainname ",
                                 "dump ",
                                 "enable ",
+                                "fpm ",
                                 "frr ",
                                 "hostname ",
                                 "ip ",
